.idea/runConfigurations.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Generally you don't need to commit any of the .idea/* files to version control. 

In your .gitignore file you could add 
``` .idea/ ```
which would ignore any future .idea files from getting committed to git

Not a big deal for this project though.